components/post-primary-category/index.js

```diff
@@ -0,0 +1,21 @@
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import { PostPrimaryTerm } from '../post-primary-term';
+
+export const PostPrimaryCategory = PostPrimaryTerm;
+
+PostPrimaryCategory.defaultProps = {
+	placeholder: __('Select a category', 'tenup'),
+};
```

Comment:
These seem duplicated with L17:L21

Reply:
Fixed in [e278bce](https://github.com/10up/block-components/pull/178/commits/e278bce9b9aabea97b5cf056e4dc4e06e09f3ced)